frontend/assets/stylesheets/components/_elevated-button.scss

```diff
@@ -0,0 +1,47 @@
+
+.elevated-button {
+    @include fs-textSans(3);
+    font-size: 20px;
+    font-weight: normal;
+
+    background-color : #005689;
+    border-color : #005689;
+
+    
+    width: gs-span(3.8);
+    
+    &:hover {
+        background-color: #003557;
+        border-color: #003557;
```

Comment:
These colours don't appear to be in guss. Can we find suitable replacements that are in guss, rather than introducing non-standard colours?